### PR TITLE
Create km-cycled.Rmd

### PR DESCRIPTION
**Pull request by natesheehan**
New PR re #59

### CODE REVIEW

File: vignettes/km-cycled.Rmd [hunk 1]
```diff
@@ -0,0 +1,203 @@
+---
+title: "Estimating million km's cycled per zone in England and Wales regions"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Reproducing cycling potential estimates in UK cities}
```
Comment (Robinlovelace) [inline vignettes/km-cycled.Rmd:5]: I think... updating this line to say:

```
   %\VignetteIndexEntry{Estimating million km's cycled per zone in England and Wales regions}
```

Will fix the error.

Looking good in any case!

**Review by Robinlovelace — CHANGES_REQUESTED** [review]

**Comment by Robinlovelace** [comment]
Many thanks for the update, let's hope it passes the check this time! Good to merge from your perspective @natesheehan ?

**Comment by natesheehan** [comment]
tests were passing before 🥳  have committed some text edits, happy to merge from my side! 🚲

**Comment by Robinlovelace** [comment]
> tests were passing before partying_face have committed some text edits, happy to merge from my side! bike

:tada: 

Many thanks for the persistence on this. Looking good, look forward to seeing it on the website!

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**